packages/deployment/app/cloudformation.template.yml

```diff
@@ -1098,13 +1164,123 @@ Resources:
     Properties:
       Cluster:
         Ref: CumulusECSCluster
-      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 0 {{/if}}
+      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 1 {{/if}}
       TaskDefinition:
         Ref: {{@key}}TaskDefinition
       DeploymentConfiguration:
         MaximumPercent: 100
         MinimumHealthyPercent: 0
 
+  {{@key}}ECSServiceScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    DependsOn:
+    - {{@key}}ECSService
+    Properties:
+      MaxCapacity: {{# if this.max_count}}{{this.max_count}}{{ else }} 100 {{/if}}
+      MinCapacity: {{# if this.count}}{{this.count}}{{ else }} 1 {{/if}}
+      ResourceId:
+        Fn::Join:
+          - /
+          - - service
+            - Ref: CumulusECSCluster
+            - Fn::GetAtt:
+              - {{@key}}ECSService
+              - Name
+      RoleARN: {{../this.iams.scalingRoleArn}}
+      ScalableDimension: 'ecs:service:DesiredCount'
+      ServiceNamespace: ecs
+
+  {{@key}}ScaleOutScalingPolicy:
+    Type: "AWS::ApplicationAutoScaling::ScalingPolicy"
+    Properties:
+      PolicyName: {{@key}}ScaleOutScalingPolicy
+      ScalingTargetId:
+        Ref: {{@key}}ECSServiceScalableTarget
+      # Note: SimpleScaling is not supported by ApplicationAutoScaling
+      PolicyType: StepScaling
+      StepScalingPolicyConfiguration:
+      {{#if this.service_scaling.scale_out.step_adjustment_config}}
+        {{{ToJson this.service_scaling.scale_out.step_adjustment_config}}}
+      {{else}}
+        # Scaling out only happens once per activities waiting alarm is triggered (so assumes new activities are scheduled which can't be placed).
+        AdjustmentType: PercentChangeInCapacity
+        StepAdjustments:
+        - MetricIntervalLowerBound: 0
+          ScalingAdjustment: 10
+      {{/if}}
+
+  {{@key}}ActivitiesWaitingAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+        - Ref: {{@key}}ScaleOutScalingPolicy
+      TreatMissingData: notBreaching
+      ComparisonOperator: GreaterThanThreshold
+    {{#if this.service_scaling.scale_out.evaluation_periods}}
+      EvaluationPeriods: {{ this.service_scaling.scale_out.evaluation_periods }}
+    {{else}}
+      EvaluationPeriods: 5
+    {{/if}}
+    {{#if this.service_scaling.scale_out.threshold}}
+      Threshold: {{ this.service_scaling.scale_out.threshold }}
+    {{else}}
+      Threshold: 0
+    {{/if}}
+      Metrics:
+        - Id: e1
+          Expression: m2 - m1
```

Comment:
This is a big improvement!  I have a nagging sense I can't pinpoint that it'll have something unexpected down the line, but I'm prepared to deal with that as it comes.